pkg/KubeArmorAnnotation/config/samples/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: myapp
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: myapp
+  namespace: myapp
+spec:
+  selector:
+    matchLabels:
+      app: myapp
+  template:
+    metadata:
+      annotations:
+        container.apparmor.security.beta.kubernetes.io/redisname: unconfined
```

Comment:
what is the purpose of 'unconfined'?

Reply:
left over files, removed the whole samples directory